From 3bebe598abaa490c6f0743b8fb0447e113913ac3 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Mon, 9 Dec 2013 19:37:51 +0100
Subject: [PATCH 035/118] sysprep: mention globbing in help for --delete

Followup of ed4bcb119cb908e98ecc1107dcd8b740ee2f484f.

(cherry picked from commit 1472fbd903befb179112a1de673c376ec5bf9b2f)
---
 sysprep/sysprep_operation_delete.ml | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/sysprep/sysprep_operation_delete.ml b/sysprep/sysprep_operation_delete.ml
index 59d5485..3db3f57 100644
--- a/sysprep/sysprep_operation_delete.ml
+++ b/sysprep/sysprep_operation_delete.ml
@@ -40,7 +40,12 @@ let op = {
     pod_description = Some (s_"\
 Delete specified files or directories.
 
-Use the I<--delete> option to specify a path to remove.");
+Use the I<--delete> option to specify a path to remove.
+
+You can use shell glob characters in the specified path; note that such
+metacharacters might require proper escape.  For example:
+
+ virt-sysprep --delete '/var/log/*.log'.");
     extra_args = [
       { extra_argspec = ("--delete", Arg.String add_paths, s_"path" ^ " " ^ s_"File or directory to be removed on guest");
         extra_pod_argval = Some "PATHNAME";
-- 
2.7.4

